xonsh/completer.py

```diff
@@ -201,24 +210,29 @@ def find_and_complete(self, line, idx, ctx=None):
             endidx += 1
         endidx = endidx - 1 if line[endidx] in WS else endidx
         prefix = line[begidx:endidx+1]
-        return self.complete(prefix, line, begidx, endidx, ctx=ctx), begidx, endidx
+        return (self.complete(prefix, line, begidx, endidx, ctx=ctx),
+                begidx, endidx)
 
     def _add_env(self, paths, prefix):
+        _ = self
```

Comment:
If `self` is truly not used, then this can probably be moved to the global scope.